### PR TITLE
chore(md): update with news in org

### PR DESCRIPTION
**Pull request by AugustinMauroy**
I update the documents to conform to the change in structure of the org.

### CODE REVIEW

File: guides/ROLES.md
Comment (JohnTitor) [inline guides/ROLES.md:1]: I guess this is still useful to understand Crowdin, do we have a similar one somewhere?

File: README.md [hunk 1]
```diff
@@ -66,40 +47,24 @@ languages and cultures. See [i18n API](./guides/I18N-API.md) to get started.
...
-| Yuki Okushi          | [JohnTitor](https://github.com/JohnTitor)                 | [fr4ternite](https://twitter.com/fr4ternite)             |
-| Zeke Sikelianos      | [zeke](https://github.com/zeke)                           | [zeke](https://twitter.com/zeke)                         |
-
-## Joining
-
-If you're interested in joining this group or would like to leave a question or comment for its members, please [create an issue](https://github.com/nodejs/i18n/issues/new) or submit a pull request. Keep in mind, you don't have to be a member of this group to start translating. You can read more about contributing translations in the [Getting started](./guides/GETTING_STARTED.md) guide.
+- [amiller-gh](https://github.com/amiller-gh) - Adam Miller
+- [amorist](https://github.com/amorist) - Amor
+- [AugustinMauroy](https://github.com/AugustinMauroy) - Augustin Mauroy
+- [bnb](htps://github.com/bnb) - Tierney Cyren
```
Comment (JohnTitor) [inline README.md:53]:
```suggestion
- [bnb](https://github.com/bnb) - Tierney Cyren
```

File: README.md [hunk 2]
```diff
@@ -66,40 +47,24 @@ languages and cultures. See [i18n API](./guides/I18N-API.md) to get started.
...
+- [FranzDeCopenhague](https://github.com/FranzDeCopenhague) - Franz de Copenhague
+- [JCMais](https://github.com/JCMais) - Jonathan Cardoso
+- [laosb](https://github.com/laosb) - Shibo Lyu
+- [LaurentGoderre](https://github.com/LaurentGoderre) - Laurent Goderre
+- [lukaszewczak](https://github.com/lukaszewczak) - Łukasz Szewczak
+- [maddhruv](https://github.com/maddhruv) - Dhruv Jain
+- [nschonni](https://github.com/nschonni) - Nick Schonning
+- [obensource](https://github.com/obensource) - Ben Michel
+- [ohnTitor](https://github.com/JohnTitor) - Yuki Okushi
```
Comment (JohnTitor) [inline README.md:62]:
```suggestion
- [JohnTitor](https://github.com/JohnTitor) - Yuki Okushi
```

File: README.md [hunk 3]
```diff
@@ -66,40 +47,24 @@ languages and cultures. See [i18n API](./guides/I18N-API.md) to get started.
...
+- [ryzokuken](https://github.com/ryzokuken) - Ujjwal Sharma
+- [sotayamashita](https//github.com/sotayamashita) - Sam Yamashita
```
Comment (JohnTitor) [inline README.md:64]:
```suggestion
- [sotayamashita](https://github.com/sotayamashita) - Sam Yamashita
```

File: README.md [hunk 4]
```diff
@@ -66,40 +47,24 @@ languages and cultures. See [i18n API](./guides/I18N-API.md) to get started.
...
-| Vanessa Yuen         | [vanessayuenn](https://github.com/vanessayuenn)           | [vanessayuenn](https://twitter.com/vanessayuenn)         |
-| Volkan Nazmi Metin   | [Volem](https://github.com/Volem)                         | [volemnic](https://twitter.com/volemnic)                 |
-| Wexpo Lyu            | [laosb](https://github.com/laosb)                         | [it99p](https://twitter.com/it99p)                       |
-| Yuki Okushi          | [JohnTitor](https://github.com/JohnTitor)                 | [fr4ternite](https://twitter.com/fr4ternite)             |
-| Zeke Sikelianos      | [zeke](https://github.com/zeke)                           | [zeke](https://twitter.com/zeke)                         |
-
-## Joining
-
-If you're interested in joining this group or would like to leave a question or comment for its members, please [create an issue](https://github.com/nodejs/i18n/issues/new) or submit a pull request. Keep in mind, you don't have to be a member of this group to start translating. You can read more about contributing translations in the [Getting started](./guides/GETTING_STARTED.md) guide.
+- [amiller-gh](https://github.com/amiller-gh) - Adam Miller
```
Comment (JohnTitor) [inline README.md:50]: Could you check again if the names and links are the exactly same as the previous? The current seems broken.

Comment (JohnTitor) [reply README.md:50]: (I'm not sure if we really need to reformat the list though.)

File: guides/ROLES.md
Comment (AugustinMauroy) [reply guides/ROLES.md:1]: Everything is on nodejs/nodejs.org.

File: README.md (hunk 4 above)
Comment (AugustinMauroy) [reply README.md:50]: I updated it and used an easier to use format

**Review by JohnTitor — APPROVED** [review]

**Pull request merged**